lib/cooked_post_processor.rb

```diff
@@ -201,24 +202,26 @@ def convert_to_link!(img)
     end
 
     if upload.present?
-      upload.create_thumbnail!(width, height, crop: crop)
+      unless skip_thumbnail
+        upload.create_thumbnail!(width, height, crop: crop)
 
-      each_responsive_ratio do |ratio|
-        resized_w = (width * ratio).to_i
-        resized_h = (height * ratio).to_i
+        each_responsive_ratio do |ratio|
+          resized_w = (width * ratio).to_i
+          resized_h = (height * ratio).to_i
 
-        if upload.width && resized_w <= upload.width
-          upload.create_thumbnail!(resized_w, resized_h, crop: crop)
+          if upload.width && resized_w <= upload.width
+            upload.create_thumbnail!(resized_w, resized_h, crop: crop)
+          end
         end
       end
 
       return if upload.animated?
 
       if img.ancestors(".onebox, .onebox-body").blank? && !img.classes.include?("onebox")
-        add_lightbox!(img, original_width, original_height, upload, cropped: crop)
```

Comment:
Not related to the change, but cropped was not being using in the `add_lightbox` method.